src/Apps/Order/Components/PriceSummary.tsx

```diff
@@ -0,0 +1,57 @@
+import { Serif, space } from "@artsy/palette"
+import React from "react"
+import { BorderBox } from "Styleguide/Elements/Box"
+import { Flex, FlexProps } from "Styleguide/Elements/Flex"
+import { Spacer } from "Styleguide/Elements/Spacer"
+
+interface PriceSummaryProps extends FlexProps {
+  price: string
+  shipping: string
+  tax: string
+  total: string
+}
+
+export const PriceSummary: React.SFC<PriceSummaryProps> = ({
+  price,
+  shipping,
+  tax,
+  total,
+  ...others
+}) => {
+  return (
+    <BorderBox hasSiblings responsive flexDirection="column" {...others}>
+      <Entry label="Price" value={price} />
+      <Entry label="Shipping" value={shipping} />
+      <Entry label="Tax" value={tax} />
+      <Spacer height={space(2)} />
```

Comment:
Does `<Spacer>`'s `height` prop not read directly into the theme and grab from `space`? If so that seems like a bug in styled-system 🤔

Reply:
It's not. It only grabs those values for the `space` properties which are limited to `p` and `m` values. I don't think it's a bug of system system so much as it is a nuance of how we do sizing.

Reply:
I see. Would it reduce complexity to follow existing patterns and piggy back on existing props that accept space, like 

```jsx
<Spacer mb={2} />
```

Then we could remove the import and thus some indirection.

Reply:
I changed this. Good spot!